client/components/TestManagement/queries.js

```diff
@@ -15,6 +15,15 @@ export const TEST_MANAGEMENT_PAGE_QUERY = gql`
             id
             name
         }
+        testPlans {
```

Comment:
REVIEWERS: even if we do not go with a separate Test Plan table, this is the query I would like to use.